### PR TITLE
Aim mode replacement

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
**This has now been approved for merge, although will be along with additional changes for overall balance**

1. Removes aimmode from almost all weapons (barring a few snipers, and pistols, so they have some kinda muh niche).
2. Removes IFF from mounted/tank guns (except the HSG).

3. Adds 'shoulder shooting' inspired by RU. This lets you natively shoot past friendlies that are directly adjacent to you, without needing IFF.

Again, this is not intended to be merged, but if this becomes a serious option, then obviously other balance changes would be required to make this fully realised.
## Why It's Good For The Game
**Attempt to shift gameplay from its current highly static, blob centric gameplay, to something a bit more dynamic and interesting for both sides.**

### Aim mode removal
Aimmode as a mechanic actively, and specifically, encourages marines to deathball as hard as possible, for significant defensive and offensive benefits.

This isn't much of an issue at low pop, but its efficiency scales dramatically with pop, leading to highly concentrated gameplay where 95% of the marines are crammed into a small area where they can fire 20 guns at any target.

This has the indirect effect of influencing balance around this kind of gameplay, which only leads to non optimal strategies (i.e. not blobbing) being worse. - case in point, not so long ago, marines would fairly reliably be able to fallback to FOB after the front goes poorly, and be able to mount a viable defence at fob. Now days, unless you can escape via tad, if the frontline collapses its almost 100% certain everyone is just going to wipe.

Removing most IFF/aimmode means it is no longer practical for marines to mass up in this way with infinitely beneficial scaling, forcing marines to spread out more, creating more dynamic, interesting, skirmishy type gameplay.

### Shoulder fire addition
The shoulder firing mechanic means you can still group up and apply psuedo aimmode, but the key difference is it can't scale with pop (you can spread your line of dudes wider if you have space, but you can't go deeper), and also requires some level of coordination, as opposed to just pressing the funny IFF button.

But yes, overall by design, marines will be encourages to spread out significantly more so they can actually apply their damage.

### Xeno/overall balance
Yes this PR in isolation will almost certainly be wack. Game balance has over time been designed around the assumption that xenos have to be able to fight a big fat blob of marines without being instantly vaporised. Would some xeno stuff need nerfing, or some marines stuff need buffing? Yes, almost certainly, but again, this PR is not intended for actual merging, just for ~~farming salt~~ testing the principal.
## Changelog
:cl:
balance: Aim mode removed from most guns
balance: Added shoulder firing - ALL guns can now shoot past ADJACENT team mates without penalty
/:cl:

### CODE REVIEW

**Comment by 00ze-cyclone** [comment]
Can guns that didn’t have aimmode shoulder fire?

**Comment by ilikesaying** [comment]
does this mean if someone is in front of me, i fire past them

**Comment by Tatarla** [comment]
Based

**Comment by Lumipharon** [comment]
> Can guns that didn’t have aimmode shoulder fire?

Yes.

>does this mean if someone is in front of me, i fire past them

Also yes. They have to be directly adjacent (i.e any of the 8 tiles around you) so doing it on the move would be some kinda impressive feat of coordination.

**Comment by lKiseki** [comment]
I've been asking for this, for the longest time now. This game (and CM, by extension) was not really designed with aim mode in mind. It tore balance a new one, and ever since then, we've felt its effects, which Lumi has accurately explained in his original post. I genuinely think we'd be better off without this, and that it is very much worth considering, if not outright implementing. I believe it would be a step in the right direction, _and_ worth the subsequent balance changes it would require.

**Comment by Mannybrado** [comment]
> case in point, not so long ago, marines would fairly reliably be able to fallback to FOB after the front goes poorly, and be able to mount a viable defence at fob. Now days, unless you can escape via tad, if the frontline collapses its almost 100% certain everyone is just going to wipe

This is something that I really, really, really want to see return to the game, as the back-and-forth when each team is evenly matched skill-wise is when the game is at it's best, imo.

However, I don't think this change is really going to bring this back. I do think that aim mode has slowly but surely made the game worse, and this change is very interesting, but I feel as though something else will be needed to get this aspect of the gameplay back--like maybe this change in conjunction with some kind of defense buff for marines.

Very interested to see how this plays out though. Should get a scheduled test time set up on discord for it.

**Comment by Araulius** [comment]
I might actually have incentive as SL to lead flank manuevers

**Comment by SimplySmiley123** [comment]
This sounds like it would kill all mounted weapons that aren't the HSG honestly.
Those guns do need their aim mode to not be a massive waste of time considering you can't put them at the front at all, you're gonna get ganked for it by the xenos (and very likely crusher will just delete your gun the second he spots your existance), and with this you won't put it behind marines to actually use the weapon for longer then a second.

**Comment by Mannybrado** [comment]
> This sounds like it would kill all mounted weapons that aren't the HSG honestly.
> Those guns do need their aim mode to not be a massive waste of time considering you can't put them at the front at all, you're gonna get ganked for it by the xenos (and very likely crusher will just delete your gun the second he spots your existance), and with this you won't put it behind marines to actually use the weapon for longer then a second.

I agree with the MG-27 specifically. Using it is a lot riskier and has some amount of downside compared to the other machine guns, and also marines get a limited amount of them, so it keeping aim mode and being able to spam as it's niche would be fair imo.

**Comment by yyzsong** [comment]
IMO this change would be complimented by a reduction in the nukewar xeno mult. If we wanna discourage blobbing on the marines' side, we want to break up xeno numbers as well.

**Comment by Arturlang** [comment]
> IMO this change would be complimented by a reduction in the nukewar xeno mult. If we wanna discourage blobbing on the marines' side, we want to break up xeno numbers as well.
> 
> > Yes.
> 
> Do all guns (eg. RR) that didn't have aimmode shoulder fire?

Yea i'm not sure how much this would make sense to allow it to happen for stuff like shotguns

**Comment by despong** [comment]
tank chaingun friendly fire will be absolute peakage

**Comment by SauceCat9** [comment]
This is a fascinating change and I've avoided commenting about it cause I'm still thinking about it, but a few things would probably have to happen to make this walanced, such as cades being stronger, xenos being reduced in number/stats, and a lot of tweaks to certain guns that were balanced around the idea of blobs. I know you've already stated that changes will have to be made to make this work, I've just been brainstorming on what this could do. I like it a lot because it encourages good positioning and removes one of the more disliked marine aspects while not fully removing playstyles or nerfing marines outright, and I'm very interested to see what a TM of this looks like.

One issue I see with it right now to be honest are that keeping within one tile of a marine you want to shoot past while moving could be very hard, but that might not be an issue at all tbh, just something I need to adjust to.

**Comment by Wisemonster** [comment]
Doesn't this make the mech and tank/apc FF magnets:? They were given the ability to be IFF'ed due to being massive front-line vehicles that block movement. This especially fucks with the tank, considering it has to be in the front to avoid blowing up other marines, but this change makes it so now it acts as wall againt your teammates gunfire.

**Comment by FosterGrime-Heymynameslouie** [comment]
testing the principal

**Comment by Looking4bros** [comment]
Based, Speedmerge.

> not so long ago, marines would fairly reliably be able to fallback to FOB after the front goes poorly

Engineering structures are too weak, i used to primarily play engineer until everything i made disintegrated as fast as a hivelords maze to PC engineers. Because metal isn't as plentiful as plasma i ceased my actions in that area. It's the strongest draw for me to go vali(besides schizophrenia), as the living flesh of a H Armor valitard is stronger than anything less than a heavy Reinforced wall. 

>This sounds like it would kill all mounted weapons that aren't the HSG honestly.

Have you ever used the HMG? It's ridiculously powerful, literally melts every xeno and i've killed a king solo when it got too cocky and pushed me while i was using it alone. If you're defending flanks and setting up fallback positions you don't need aim mode and the true power of these guns gets utilized. Can you think of any time where you didn't use the MG-27 on aim mode? Doubling your firerate is nothing to sneeze at.

>No matter what the deathball is gonna stay till smaller squads are both given some space to be viable at all and not throwing free kills at xenos

Small group survivability is what Vali is all about, become the main character.

**Review by Kuro020 — APPROVED** [review]
Lgtm for a tm

**Comment by SimplySmiley123** [comment]
> Have you ever used the HMG? It's ridiculously powerful, literally melts every xeno and i've killed a king solo when it got too cocky and pushed me while i was using it alone. If you're defending flanks and setting up fallback positions you don't need aim mode and the true power of these guns gets utilized. Can you think of any time where you didn't use the MG-27 on aim mode? Doubling your firerate is nothing to sneeze at.

Tried the HMG, every single time a lone t3 xeno would run up if there wasn't a deathball of marines between me and them and either just kill me, leaving the HMG to the whims of the other marines who would often just, abandon it entirely because of it being a massive hussle to drag around, or it gets the acid or crusher treatment.
And you do kinda need that aim mode for the fallback positions to not gun down the retreating marines. otherwise, you need it for the real protection from getting all your hopes and dreams ruined by a crusher, a wall of meat shields - marines.

> Small group survivability is what Vali is all about, become the main character.

Not really?
Vali gets cucked lately from what I got to see and experience. You're gonna be in the face of a xeno who can just, use a stun, or fish you out and take you to the shadow realm, chicago.
You're outright worse then a John Newbie walking around with nothing but ar12 at best and an outright liability to other marines at worst.

**Comment by Pariah919** [comment]
(i'm supposed to doomer here so insert doomer paragraph?)

Either way, I'll be interested to see what exactly is proven from this I doubt its anything I won't expect. 
Deathballs have always been the way to go even before aim mode was a thing. There is no incentive to spread out when spreading out generally means the opposing deathball can just pick you off squad by squad. Xenos are far faster and tougher than you so skirmishing with them is usually a fools errand since a losing skirmish they can usually run away from while they'll usually run you down. Alongside Nuclear War's mechanics explicitly encouraging deathballing and quikbild encouraging it even more due to the entire map being prepped with 2x2 death mazes by the time you're landing.

**Comment by BraveMole** [comment]
Based

**Comment by SimplySmiley123** [comment]
> (i'm supposed to doomer here so insert doomer paragraph?)
> 
> Either way, I'll be interested to see what exactly is proven from this I doubt its anything I won't expect. Deathballs have always been the way to go even before aim mode was a thing. There is no incentive to spread out when spreading out generally means the opposing deathball can just pick you off squad by squad. Xenos are far faster and tougher than you so skirmishing with them is usually a fools errand since a losing skirmish they can usually run away from while they'll usually run you down. Alongside Nuclear War's mechanics explicitly encouraging deathballing and quikbild encouraging it even more due to the entire map being prepped with 2x2 death mazes by the time you're landing. Also, FOB defenses working was mostly hives being godawful at breaking through FOB's somehow. Distress mechnically encouraged you to go to an instant FOB defence if you failed to attack a position, there was no reason to actually fight people. We've made it so you can easily swap your entire hive comp in like 3-5 minutes to spec into killing FOBs, so no wonder FOBs are so easy to break huh?
> 
> I don't think reverting to pre aim mode will suddenly make people go to squad level tactics, if anything it'll probably encourage dballing even more since spreading out thinly is discouraged since you'll get shrekt by 2 rounies in your small group of 4 wasd'ing around you. That's by design. You can't 'fix' that unless you make all the backliners properly squishy and start shooting things like droppods.
> 
> Also removing IFF from all the mounted guns basically kills them in comparison to their non-mounted counterparts lel. Just remove them from play entirely since that's like... 90% of the benefit you get in exchange for being entirely static. There's a reason nobody really runs the HMG outside of specific loadouts or situations.
> 
> tl;dr I think its laughable to believe that aim mode is the reason people death stack, both teams are in constant huge death stacks. The team that is slower is inheriently forced to death stack more, because they can't really run away and rely on their numbers to keep assaults up and going.

Yeahhh... Marines at best have the Tadpole, I guess, for splitting off? And even then, xenos can and will overrun it nigh instantly if they want the guaranteed round removal kills. 
Xenos have the speed.
Xenos have teleports around the map in form of the tunnels.
Night vision.
X-ray vision.
Stun abilities galore.
Quick swapping castes to quickly readapt to whatever's causing trouble (tho only if the xeno really cares about it).

While marines have the aim mode.
And maybe CAS once in a blue moon doing anything of value.
Tank I guess when it's not busy vibe checking marines and didn't already die when driver fell asleep behind the wheel.
Had the tadpole before it got relegated to being the mobile req beacon in the back.

**Comment by zzurge** [comment]
It'll be a hard crutch to get off of using but i think itll be a change for the better in time,in the interim Smartgunners will be more relevant again which is good news to me

**Comment by MiniMeatwad** [comment]
Nerf xenos so people think marines are better without aim mode. Genius I think

**Comment by MiniMeatwad** [comment]
> they'll just blame the person for moving if they were doing shoulder fire.
Wear HOD

**Comment by blackdav123** [comment]
only the first projectile fired is IFF so buckshot will still hit everything but the main pellet

**Comment by jroinc** [comment]
> If this gets explored in more detail after the test I'd say there would have to be some negative reinforcement for friendly fire, as currently the punishment is for the person being shot. Which... if someone is being careless they're not going to give a shit if they just put twenty rounds into the back of a friend as they're used to flailing on aim mode, and ofc they'll just blame the person for moving if they were doing shoulder fire.

There's currently a deactivated automod system for this that only kicks.

Activating it, adding actual penalties, and lowering the values shouldn't be too hard; something like the first X amount of damage is free, damage between X and Y deals a message and a stack of stagger, damage between Y and Z deals a harsher message and a 0.1 tick stun (to force-drop/unwield weapons).

**Comment by jroinc** [comment]
https://github.com/tgstation/TerraGov-Marine-Corps/blob/master/code/modules/mob/living/living_helpers.dm

from line 47 down

**Comment by Araulius** [comment]
scout rifle REMAINS UNDEFEATED

**Comment by Looking4bros** [comment]
Tested in Valhalla, Underbarrel shotguns have the shoulder fire IFF and can hit xenos on the same tile as a friendly with buck/flechette/slug. normal shotguns cannot unless they're slugs.

**Comment by xSlumLord** [comment]
It's hard to give any conclusive feedback considering that it's currently just a straight xeno buff considering nothing else was tweaked. Xeno count could be perhaps lowered to adjust for no ungaballing, but I'm having a hard time telling if this is a healthy change without seeing other aspects tweaked to match.

**Comment by Looking4bros** [comment]
if being thrown off tank/APC when you don't have a hand free was removed, you wouldn't need IFF, just the will to jump on, because you won't just eat wall/dirt for trying. That mechanic only hurts marines, for HVH i don't know or really care.

**Comment by Lumipharon** [comment]
> if being thrown off tank/APC when you don't have a hand free was removed, you wouldn't need IFF, just the will to jump on, because you won't just eat wall/dirt for trying. That mechanic only hurts marines, for HVH i don't know or really care.

U wotm8?

**Comment by Looking4bros** [comment]
> U wotm8?

I don't care much for HVH, i thought Campaign was going to be multi map Xeno fighting.

but if u wotting me about the tank stuff, it's a complaint that was brought up, that "muh no aim mode means no tank defense" is only a problem because marines can't fight wielded on tank.

**Comment by Looking4bros** [comment]
Comedic idea of removing SG skill with the removal of aim mode. Truly one step closer to the removal of specs.

**Comment by SimplySmiley123** [comment]
> 
> Genuine skill issue in both regards. Use fortifications for the HMG and offset yourself, even if you have to pick a spot on the other side of the push. Vali is mostly just number maxxxing. Claymore is the only real big xeno fighter, if you're trying to skirmish instead of brawl, you can go spear. And people using the machete as a primary are fools, it just sucks so much in comparison.
> 
> Ran HMG first test round and the round prior. works as intended, it's fine.

You do realise fortifications suck utter, unapologetic ass currently?
They can stop a runner or a drone long enough to fend them off but anything bigger treats those as a speed bump.
Only really sandbags aren't complete trash solely because of how spammable they are by regular marines.
Plus, wherever you have it pointed at, is a direction marines can't push into. You mow marines down harder with it then xenos ever.

**Comment by Lumipharon** [comment]
> > U wotm8?
> 
> I don't care much for HVH, i thought Campaign was going to be multi map Xeno fighting.
> 
> but if u wotting me about the tank stuff, it's a complaint that was brought up, that "muh no aim mode means no tank defense" is only a problem because marines can't fight wielded on tank.

No, I genuinely don't know what you said.

But you can shoot xenos on tanks without aimmode/IFF. Just need to click the sprite.

**Comment by Lauwma** [comment]
Just remove aimmode on almost all guns, strip most iff and give more damage in compensation 🤷

**Comment by iforgotmeword** [comment]
:/
Off topic but 
If we wanted to do squad gameplay you would have to limit xenos Regen to a cap in each region 

EX zones have a cap of 400 healing points 
If there is 24 xenos in zone one they can only heal so fast
So if the xenos split up they would maximize healing overall

Also power should be moved to LZ and make so you can't replace battery's if you want FOB to of use (note caves is what I personally think is kills FOB more then anything else)

**Comment by breadhunt** [comment]
I think this is a good change in the long run. Squad-focused stuff and teamwork is where TGMC really shines since it allows for so much more variety in gameplay - any step towards that and away from boring, static deathballing is a good step. The shoulder shooting sounds cool as fuck too, I wonder if huddling up to gain IFF will become a strat for squads to counter runners and the like. 

I think it will be really important, beyond just making small-squad stuff viable, to also have actual objectives for squads. There’s a few we have currently - covering backlines, researcher digsites, intel computer runs are fun.. But most of these are situational and usually all of them can be accomplished by one squad, if that. Some, like miner holds, are difficult to balance without becoming frustrating (either xenos are at a stalemate, or the miner force wipes completely - no in between or chance for recovery). 

I play a lot of SL, and the hardest part of trying to make the fun, dynamic, risky squad stuff work - the BEST part of the game, the tactics, the community, the part that TGMC should (imo) revolve around - ***isn’t*** “how can we make [x] plan work?”. 

It’s, “uhh, guys, any ideas for what we should do today?” That's not a balance issue, that's not a "ooh xenos too strong", that is a fundamental design issue with how objectives are structured in TGMC that needs to be addressed, ideally through actual dynamic objectives instead of just adding more places to sit around and wait for bars to go up.

Right now, we don’t actually *have* objectives that demand cohesion and squad tactics. Marines need reasons to split up and take space, to tackle objectives from different angles. Obviously making small squad stuff viable is nice and will help promote teamwork. But if this gives more reasons to split up (i.e reduction in FF) then I **absolutely** think removing aimmode is worth working through any balance issues, hands down, because at the minute there is zero reason to strategise and work together other than literal fucking novelty.

**Comment by jroinc** [comment]
Would it be possible to allow bipods to access aimmode as their big thing? It *requires* being stationary.

**Comment by Lauwma** [comment]
> I think this is a good change in the long run. Squad-focused stuff and teamwork is where TGMC really shines since it allows for so much more variety in gameplay - any step towards that and away from boring, static deathballing is a good step. The shoulder shooting sounds cool as fuck too, I wonder if huddling up to gain IFF will become a strat for squads to counter runners and the like. 
> 
> 
> 
> I think it will be really important, beyond just making small-squad stuff viable, to also have actual objectives for squads. There’s a few we have currently - covering backlines, researcher digsites, intel computer runs are fun.. But most of these are situational and usually all of them can be accomplished by one squad, if that. Some, like miner holds, are difficult to balance without becoming frustrating (either xenos are at a stalemate, or the miner force wipes completely - no in between or chance for recovery). 
> 
> 
> 
> I play a lot of SL, and the hardest part of trying to make the fun, dynamic, risky squad stuff work - the BEST part of the game, the tactics, the community, the part that TGMC should (imo) revolve around - ***isn’t*** “how can we make [x] plan work?”. It’s, “uhh, guys, any ideas for what we should do today?” 
> 
> 
> 
> That's not a balance issue, that's not a "ooh xenos too strong", that is a fundamental design issue with how objectives are structured in TGMC that needs to be addressed, ideally through actual dynamic objectives instead of just adding more places to sit around and wait for bars to go up. That's kind of bad; no amount of quick-fix number changes will change that. 
> 
> 
> 
> _(Ooh, yeah guys, let's make marines have to defend the FOB! You know, the place that xenos have literally 0 reason to give a shit about unless they're actively sieging it? That FOB? Because SS13 and by extension CM is a fucking master class on game design that we should copy from?)_ 
> 
> 
> 
> Right now, we don’t actually *have* objectives that **reward** cohesion and squad tactics, let alone demand them. Marines need reasons to split up and take space, to tackle objectives from different angles. Obviously making small squad stuff viable is nice and will help promote teamwork. But if this gives more reasons to split up (i.e reduction in FF) then I **absolutely** think removing aimmode is worth working through any balance issues, hands down, because at the minute there is zero reason to strategise and work together other than literal novelty.

Before having objectives we needed to have squad viability in the walance. Aimmode removal is the obligatory step toward that

**Comment by SimplySmiley123** [comment]
> I think this is a good change in the long run. Squad-focused stuff and teamwork is where TGMC really shines since it allows for so much more variety in gameplay - any step towards that and away from boring, static deathballing is a good step. The shoulder shooting sounds cool as fuck too, I wonder if huddling up to gain IFF will become a strat for squads to counter runners and the like.
> 
> I think it will be really important, beyond just making small-squad stuff viable, to also have actual objectives for squads. There’s a few we have currently - covering backlines, researcher digsites, intel computer runs are fun.. But most of these are situational and usually all of them can be accomplished by one squad, if that. Some, like miner holds, are difficult to balance without becoming frustrating (either xenos are at a stalemate, or the miner force wipes completely - no in between or chance for recovery).
> 
> I play a lot of SL, and the hardest part of trying to make the fun, dynamic, risky squad stuff work - the BEST part of the game, the tactics, the community, the part that TGMC should (imo) revolve around - _**isn’t**_ “how can we make [x] plan work?”. It’s, “uhh, guys, any ideas for what we should do today?”
> 
> That's not a balance issue, that's not a "ooh xenos too strong", that is a fundamental design issue with how objectives are structured in TGMC that needs to be addressed, ideally through actual dynamic objectives instead of just adding more places to sit around and wait for bars to go up. That's kind of bad; no amount of quick-fix number changes will change that.
> 
> _(Ooh, yeah guys, let's make marines have to defend the FOB! You know, the place that xenos have literally 0 reason to give a shit about unless they're actively sieging it? That FOB? Because SS13 and by extension CM is a fucking master class on game design that we should copy from?)_
> 
> Right now, we don’t actually _have_ objectives that **reward** cohesion and squad tactics, let alone demand them. Marines need reasons to split up and take space, to tackle objectives from different angles. Obviously making small squad stuff viable is nice and will help promote teamwork. But if this gives more reasons to split up (i.e reduction in FF) then I **absolutely** think removing aimmode is worth working through any balance issues, hands down, because at the minute there is zero reason to strategise and work together other than literal novelty.

Well this PR alone wouldn't do shit to getting rid of the deathballs. It needs more done both for objectives forthe marines and honestly everything about xenomorphs would have to change to work with squads splitting off.
It needs more then one step to not be just an overall nerf across the board right after xenos got buffed al lacross the board already.
Marines got used to it but xenos still just absolutely shaft any idea of _not_ deathballing as marine.
They are too healthy, can pull out stuns from the ass to outright nullify anything that isn't a massive horde of marines reliably and consistently, got freedom over the map and can get anywhere the main ungaball isn't at, alongside being able to change caste on the fly to fit the situation better, having gargoyles to notify them immediately of anything and anyone away from the main marine force to focus down for free guaranteed kills, nothing stops xenomorphs from deathballing a small squad ignoring the mainforce for a moment.

Plus the way the objectives are.
Both Crash and Nukewar. You can only feasibly do one disk at a time unless xenomorphs threw the round entirely and are down to last 2-3 xenos left with no silo, no tier 4s, no nothing. You can't really go off to do miners without focusing the entire round on the miners or instead be lucky and have a round on a map where a miner is close enough to an objective to have the protection of the deathball.

On top of immense risk when you're not in the main ungaball - the risk of getting effortlessly removed from the round entirely. If someone goess off, they do this wanting to try and go out with a bang before going to play something else entirely, everyone else stays with the ball, because that's the only way they don'trisk getting forced out of the game with nothing to do about it.

**Comment by SimplySmiley123** [comment]
> Before having objectives we needed to have squad viability in the walance. Aimmode removal is the obligatory step toward that

You also need the second, maybe third step for the first step to not be absolute bollocks.
There needs to be a guarantee otherwise this does absolutely nothing to encourage squad gameplay.

You need the objectives that reward it.
You need it to be viable to begin with and it just isn't right now.

**Comment by Lumipharon** [comment]
Uh, what does that have to do with this pr?

**Comment by Looking4bros** [comment]
I was not aware that you could click on the sprite of the xeno to shoot them while they're on the tank without IFF, as i do not use guns besides memeing.  It would let marines jump onto the tank with a wielded weapon instead of clicking the xeno, easier to miss into a wall than the tank/APC.

**Comment by SimplySmiley123** [comment]
Yeah no marines currently can't do jack shit.

Xenos are too strong at wiping out separated marines with no effort.
Vali, shotgun users, smg users, AR18 users exist and will always be too far in the front for shoulder fire to ever matter.
Marines are still encouraged to constantly move and shat on if they stay too long in one place trying to coordinate the shoulder fire.
Xenos on top of an APC and tank can't be touched for most of the time they're on those thanks to their sprites disapeparing after the vehicle at random.
Mounted weapons that aren't the Tadpole HSG and the marine vendor HSG are nigh useless.

It is a good step, but by god it needs more changes to not be insufferable and actually end up with marines not blobbing together.

**Comment by breadhunt** [comment]
Having played a few rounds of this...

> Well this PR alone wouldn't do shit to getting rid of the deathballs. 

Yeah we need good (ATTACKING!!) objectives, and ideally more ways of marines recovering/mitigating from the loss of a wipe. I hope we get more of that in the future.

But holy shit, removing aimmode has been _really_ effective at buffing chokepoints. To get past chokepoints you now actually have to coordinate support fire or flank, whereas previously chokepoints were countered by just aimballing. This is cool, it poses a problem which marines can only solve through timing an attack or taking more space.

+1 me like

**Comment by SimplySmiley123** [comment]
> But holy shit, removing aimmode has been _really_ effective at buffing chokepoints. To get past chokepoints you now actually have to coordinate support fire or flank, whereas previously chokepoints were countered by just aimballing. This is cool, it poses a problem which marines can only solve through timing an attack or splitting up and taking more space.

Which boils down to spamming out OB at every single choke.
Mortar and Howi are good enough at area denial to both sides, but rarely if at all help with pushing.

Or spamming DMRs to try and at least shoot something while vali users are too busy being full cover for xenomorphs.

**Comment by breadhunt** [comment]
> Which boils down to spamming out OB at every single choke. Mortar and Howi are good enough at area denial to both sides, but rarely if at all help with pushing.

Separate problem but yes, OB is one of the best tools marines have for coordinating pushes (clear announcement, reason to care, reasonable chance of success)

> 
> Or spamming DMRs to try and at least shoot something while vali users are too busy being full cover for xenomorphs.

..this sounds fine? They are a support weapon after all

**Comment by Fawcksy** [comment]
> Having played a few rounds of this...
> 
> > Well this PR alone wouldn't do shit to getting rid of the deathballs.
> 
> Yeah we need good (ATTACKING!!) objectives, and ideally more ways of marines recovering/mitigating from the loss of a wipe. I hope we get more of that in the future.
> 
> But holy shit, removing aimmode has been _really_ effective at buffing chokepoints. To get past chokepoints you now actually have to coordinate support fire or flank, whereas previously chokepoints were countered by just aimballing. This is cool, it poses a problem which marines can only solve through timing an attack or splitting up and taking more space.
> 
> +1 me like

personally think it's better just to end the round than to have more wipe recovery, best recovery from a bad wipe would be a fresh start

**Comment by Churchtuary** [comment]
There is nothing in this PR that has any form of good will behind it and it's sole purpose is to directly hinder a team that already has to deal with micromanaging resources (ammo, health, terrain, manpower) when so many marine players rely on a close combat build who are now made essentially useless. Marines already experienced absurd steamroll scenarios where all xenos have to do is a single flanking maneuver on their backline. Chokepoints now essentially become marine death zones. Even with aim mode, chokes were already a nightmare to push into since xenos can simply take cover behind walls and retaliate with either defiler or boiler gas. Friendly fire incidents were already a major factor for marine losses, specially against melee, shotgun, laser and smg users. MG-60 and MG-42 players now block an entire hallway from panic firing since they can also no longer aim. MG-42 has been pretty much rendered useless. Maneuvering around a firing line also takes precious time that a xeno can use to scurry away, where said xeno wouldn't have pushed into that position if it was under threat of aim mode fire.

There are dozens of scenarios where marines without aim mode would have a massive disadvantage against xenos and there's even more scenarios where the heavy use of aim mode also didn't make a difference for marine victory.

Lastly, there's no data _presented_, _that I know of_, that shows why this change was even necessary in the first place.

**Comment by Neskah** [comment]
this seems like a terrible idea without rebalancing every gun that currently exists in the game, i guess you're gonna be busy

**Comment by Looking4bros** [comment]
>MG-42 has been pretty much rendered useless.

If the HMG isn't useless, the MG-42 isn't useless. MG-42 trades power for convenience, roughly 2/3rds the DPS for ease of use. You just have to pack sandbags and actually pick places to hold. It's the same difference between the mortar and the Howitzer. The biggest boon for MG-42 over HMG is that if you HAVE to take a spot that can be crusher charged, you can move it fast without being forced to try and push it somewhere safe, and you can in a pinch wield it in hand to spray and pray from your nest at whatever got inside.

**Comment by SimplySmiley123** [comment]
> ..this sounds fine? They are a support weapon after all

I dunno if it's all that fine.
It makes using any guns that aren't valitard weapons, shotguns and ar18 a _chore_ with only viable option 99% of the time being the DMR and just the DMR.
It's no longer just a suport weapon 1 or 2 marines pick but the _only_ weapon viable at anywhere but right in the xeno's face.

**Comment by SimplySmiley123** [comment]
> > MG-42 has been pretty much rendered useless.
> 
> If the HMG isn't useless, the MG-42 isn't useless. MG-42 trades power for convenience, roughly 2/3rds the DPS for ease of use. You just have to pack sandbags and actually pick places to hold. It's the same difference between the mortar and the Howitzer. The biggest boon for MG-42 over HMG is that if you HAVE to take a spot that can be crusher charged, you can move it fast without being forced to try and push it somewhere safe, and you can in a pinch wield it in hand to spray and pray from your nest at whatever got inside.

No one uses the HMG for anything but wall removal if there's no PC engies.
Or taking it to solo miner in the middle of nowhere because they already don't care about the non-existant marines near them.

It is fucking useless in every other scenario involving more then one marine in the area.

**Comment by Notamaniac** [comment]
> And even then all it takes is a single crusher to ruin your hopes and dreams if you also didn't pack 4 fucking roundstart sentries to defend just you.

https://github.com/user-attachments/assets/e7fca87a-1d9c-49c4-b8a9-36ce320650f0

**Comment by Fawcksy** [comment]
Put aim mode back on the martini mother fucker!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by iforgotmeword** [comment]
> > And even then all it takes is a single crusher to ruin your hopes and dreams if you also didn't pack 4 fucking roundstart sentries to defend just you.
> 
>  Sandbags.1.mp4

What you didn't show is a warrior ganking you as you try and place down sandbags and then boiler forcing you off your gun while marines bilnd fire your MG

**Comment by MLGTASTICa** [comment]
Finnaly , i mentioned that aim mod was a bad fix the moment it was merged... TGMC is so back...

**Comment by SimplySmiley123** [comment]
> > > And even then all it takes is a single crusher to ruin your hopes and dreams if you also didn't pack 4 fucking roundstart sentries to defend just you.
> > 
> > 
> > Sandbags.1.mp4
> 
> What you didn't show is a warrior ganking you as you try and place down sandbags and then boiler forcing you off your gun while marines bilnd fire your MG

Or marines fluoride staring at you while a sentinel waltzes up to you, watch you die, watch the bean escape and only then start firing at the general direction of the escapee, taking a year to bring you to a corpsman and leaving your MG behind to get smacked by a random drone passing by to wall off where marines came from.

**Comment by chaosvolt** [comment]
I won't weigh in on this change myself, but future advice to contributors from someone who helps with a different project: if you're actually serious about not wanting a PR to be merged prematurely, it's better to open that PR as a draft instead of saying "not ready for merging" or trusting a label to prevent it.

If merge bait was the intent then uh, not my circus so not my monkeys.

**Comment by Liznerd** [comment]
So... this PR was made to be ready for implementation (which already got implemented) but is also a suggestion, which requires a WHOLE rebalance to the whole game, while also not including the rebalance needed nor anything near equal to what is being lost, just the nerf part of the game...? I see you had the best intentions and felt like this was a healthy change to the game, but this is one thing that set us apart from CM, the machineguns and support weapons were made with aim mode in mind, the guns as of right now, don't have the damage needed to even make up for the loss of fire volume, in short, nothing was done to balance the loss this PR brought to the game to this day.
